packages/sample-controllers/src/sample-gas-prices-controller.test.ts

```diff
@@ -300,8 +300,11 @@ describe('SampleGasPricesController', () => {
  * required by the controller under test.
  */
 type RootMessenger = Messenger<
-  ExtractAvailableAction<SampleGasPricesControllerMessenger>,
-  ExtractAvailableEvent<SampleGasPricesControllerMessenger>
+  // Use `string` rather than 'Root' here to allow registering actions and publishing events from
```

Comment:
Oh that's a good idea, I hadn't thought of that.

Reply:
Alright, it works now.